src/styles/main.scss

```diff
@@ -1,3 +1,40 @@
+@import "utils/variables";
+
 body {
-  margin: 0;
+  font-size: 30px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding-top: 50px;
+}
+
+.calendar {
```

Comment:
Create a separate for calendar block
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/0f0156b0-9694-4003-887e-7634171b04f6)